tests/pipelines/test_pipelines_image_to_text.py

```diff
@@ -257,14 +257,16 @@ def test_large_model_tf(self):
     @require_torch
     def test_conditional_generation_llava(self):
         pipe = pipeline("image-to-text", model="llava-hf/bakLlava-v1-hf")
-        url = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/tasks/ai2d-demo.jpg"
-        image = Image.open(requests.get(url, stream=True).raw)
 
         prompt = (
             "<image>\nUSER: What does the label 15 represent? (1) lava (2) core (3) tunnel (4) ash cloud?\nASSISTANT:"
         )
 
-        outputs = pipe(image, prompt=prompt, generate_kwargs={"max_new_tokens": 200})
+        outputs = pipe(
+            "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/tasks/ai2d-demo.jpg",
+            prompt=prompt,
+            generate_kwargs={"max_new_tokens": 200},
+        )
```

Comment:
may I know why this new part is relevant for this PR?

Reply:
The more tests the better I assume :)

Reply:
I'm also a little unclear - doesn't this mean this test was failing before as `outputs` wasn't defined?